boards/communication/src/communication.rs

```diff
@@ -266,9 +263,7 @@ impl RadioManager {
             mav_sequence: 0,
         }
     }
-    pub fn send_message(&mut self, m: Message) -> Result<(), HydraError> {
-        let payload: Vec<u8, 255> = postcard::to_vec(&m)?;
-
+    pub fn send_message(&mut self, payload: Vec<u8, 255>) -> Result<(), HydraError> {
```

Comment:
Could also create another another send function that takes just a Message type and matches it to the needed size defined by the consts in the messages library. We should have a way to send messages that isn't stuffing.